randovania/patching/prime/all_prime_dol_patches.py

```diff
@@ -78,13 +93,21 @@ def remote_execution_patch_start() -> list[BaseInstruction]:
         *return_code,
     ]
 
-    num_bytes_to_invalidate = _remote_execution_max_byte_count - assembler.byte_count(intro)
+    num_bytes_to_invalidate = _remote_execution_max_byte_count - (assembler.byte_count(cutscene_check) + 8 +
```

Comment:
If you add the cutscene_check directly into `intro`, this line won't need to be changed